srcDLL/DllMain.cpp

```diff
@@ -85,16 +83,15 @@ int TApp::Init()
 	// Order of precedence for loading vol files is:
 	// ART_PATH (from console module), Console Module, Ini Modules, Addon directory, Game directory
 
-	// Load command line modules
-	consoleModuleLoader->LoadModules();
-
 	// Load all active modules from the .ini file
 	moduleLoader->LoadModules();
 
 	// Find VOL files from additional folders
-	for (std::size_t i = 0; i < consoleModuleLoader->Count(); ++i) {
-		// ConsoleModule name matches relative folder from game exeucutable folder
-		LocateVolFiles(consoleModuleLoader->GetModuleName(i));
+	for (std::size_t i = 0; i < moduleLoader->Count(); ++i) {
+		std::string moduleDirectory = moduleLoader->GetModuleDirectory(i);
+		if (moduleDirectory != std::string()) {
+			LocateVolFiles();
```

Comment:
Hmm, this call lost the parameter for the console module folder. The old corresponding line was:
```cpp
LocateVolFiles(consoleModuleLoader->GetModuleName(i));
```

Although tempting, it is not possible to use `moduleLoader->GetModuleDirectory(i)` here, since that returns an absolute path, whereas the `LocateVolFiles` function only accepts relative paths.

For now, I think this needs to be updated to:
```cpp
LocateVolFiles(moduleLoader->GetModuleName(i));
```